src/content/posts/2022/bootstrap-5-2-2.md

```diff
@@ -0,0 +1,46 @@
+---
+author: mdo
+date: "2022-10-03T16:00:00Z"
+title: Bootstrap 5.2.2
+keywords:
+  - bootstrap
+  - release
+---
+
+Bootstrap v5.2.2 has landed with new bug fixes and documentation updates—keep reading to see what's changed!
+
+## Highlights
+
+- **Accordion**
+  - Use Sass variable for the accordion color instead of an invalid CSS variable
+- **Buttons**
+  - Undo changes to `.btn:hover` from v5.2.1. We now explicitly target `.btn-check` styles instead.
+- **Tables**
+  - Don't redefine `$border-color` in `table-variant()` mixin
+- **Dropdowns**
+  - Temporarily restore ability for dropdowns to work without an explicit `data` atttribute (will be removed again in v6)
+- **Modals**
+  - Improve modal event listeners
+  - Use `<h1>` for all `.modal-title` instances in our docs
+- **Tabss**
+  - Ensure Tab plugin has proper keyboard functionality
```

Comment:
This one worked well on v5.2.1. It has been broken during the development of the v5.2.2. So IMO it shouldn't appear in this list.